staging/src/k8s.io/apiserver/pkg/endpoints/handlers/patch.go

```diff
@@ -86,21 +86,21 @@ func strategicPatchObject(
 	patchJS []byte,
 	objToUpdate runtime.Object,
 	versionedObj runtime.Object,
-) (originalObjMap map[string]interface{}, patchMap map[string]interface{}, retErr error) {
-	originalObjMap = make(map[string]interface{})
+) error {
```

Comment:
I just removed the return values here since they are unsuitable for use after having used them to apply the patch